src/components/component-anchor-nft.js

```diff
@@ -11,29 +11,53 @@ AFRAME.registerComponent('arnft-anchor', {
   },
   init: function () {
     var _this = this
-    console.log(this.data)
+    // not sure if this 'container' is needed
```

Comment:
I think this part of the code is yet included in **_startWorker** function, probably is not needed...